### DISCUSSION TITLE
How can you change the IP address of the server that runs Stable diffusion

### DISCUSSION THREAD

**Discussion by BlGhost8309** [lllyasviel/Fooocus#2166] [Q&A] [answered]
Hello.  Please tell me how to configure SD Fooocus so that it does not start on the local host, but uses the ip that I specify.  The thing is that I want to run SD on a cloud server and I want to be able to access it from my computer.  Thank you.

**Comment by mashb1t** [comment, marked as answer]
With `--listen` you can specify an IP to attach to. Without any further arguments it binds on 0.0.0.0 (accepts incoming connections from all NICs), but you can bind to a specific IP / NIC by using `-- listen 123.123.123.123`
